charts/gardener-extension-admission-aws/charts/application/templates/validatingwebhook-validator.yaml

```diff
@@ -17,7 +17,9 @@ webhooks:
     - cloudprofiles
     - shoots
   failurePolicy: Fail
-  objectSelector: {}
+  objectSelector:
```

Comment:
It should be configurable via the Helm chart values whether the object selector should be used. Otherwise, this might not be backwards-compatible if the Gardener version is still old (not adding the labels) while the admission webhooks already relys on them.

Same for other extensions, of course.